app/src/main/java/com/codesoom/assignment/TaskHttpHandler.java

```diff
@@ -0,0 +1,188 @@
+    private final int CREATED = 201;
+    private final int DELETED = 204;
+    private final int NOT_FOUND = 404;
+
+    @Override
+    public void handle(HttpExchange exchange) throws IOException {
+        String method = exchange.getRequestMethod();
+        URI requestURI = exchange.getRequestURI();
+        String path = requestURI.getPath();
+
+        InputStream inputStream = exchange.getRequestBody();
+        String body = new BufferedReader(new InputStreamReader(inputStream))
+                .lines()
+                .collect(Collectors.joining("\n"));
+
+        if(method.equals("GET") && path.equals("/tasks")){
+            String content = taskListToJSON();
+            exchange.sendResponseHeaders(SUCCESS, content.getBytes().length);
+            sendResponseBody(exchange,content);
+        }
+
+        if(method.equals("GET") && path.startsWith("/tasks/")){
+            String searchIDString = path.substring("/tasks/".length());
+            System.out.println("ID String : " + searchIDString);
+            getRequest(exchange,searchIDString);
+        }
+
+        if(method.equals("POST") && path.equals("/tasks") && !body.isBlank()) {
+            postRequest(exchange, body);
+        }
+
+        if (method.equals("PUT") && path.startsWith("/tasks/") && !body.isBlank()) {
+            String searchIDString = path.substring("/tasks/".length());
+            System.out.println("ID String : " + searchIDString);
+            putRequest(exchange,searchIDString, body);
+        }
+
+        if (method.equals("DELETE") && path.startsWith("/tasks/")) {
+            String searchIDString = path.substring("/tasks/".length());
+            System.out.println("ID String : " + searchIDString);
+            deleteRequest(exchange, searchIDString);
+        }
+    }
+
+    private Long isValidURL(String IDString)  {
+        try {
+            Long searchID = Long.parseLong(IDString);
+            return searchID;
+        }
+        catch (Exception e){
+            throw e;
+        }
+    }
+
+    private void sendResponseBody(HttpExchange exchange, String content) throws IOException {
+        OutputStream outputStream = exchange.getResponseBody();
+        outputStream.write(content.getBytes());
+        outputStream.flush();
+        outputStream.close();
+    }
+
+    private void postRequest(HttpExchange exchange, String body) throws IOException {
+        Task task = makeTask(body);
+        task.setId(++curTaskID);
+        taskList.add(task);
+
+        String content = taskToJSON(task);
+        exchange.sendResponseHeaders(CREATED, content.getBytes().length);
+        sendResponseBody(exchange, content);
+    }
+
+    private void getRequest(HttpExchange exchange, String strID) throws IOException {
+        String content = "";
+        try{
+            Long searchID = Long.parseLong(strID);
+            int taskIdx = findTaskIdx(searchID);
+
+            content = taskToJSON(taskList.get(taskIdx));
```

Comment:
```suggestion
            final Long searchID = Long.parseLong(strID);
            content = taskToJSON(findTask(searchId);
```

대충 이런식이면 어떨까요? Task가 List 자료구조로 되어있다는 것을 굳이 알아야할 필요가 있을까 싶어서요!